Source/WebCore/platform/graphics/haiku/ShareableBitmapHaiku.cpp

```diff
@@ -71,9 +71,13 @@ void ShareableBitmap::paint(GraphicsContext& context, float scaleFactor, const I
 
 WebCore::PlatformImagePtr ShareableBitmap::createPlatformImage(WebCore::BackingStoreCopy, WebCore::ShouldInterpolate)
 {
+    // FIXME: This doesn't work with UNIX's implementation of shared memory atm.
```

Comment:
>Do we get this far in the execution of WebKit yet?

For some reason, no. (Next problem to investigate?)

> Is there a reason SharedMemoryHaiku cannot be used? What's the problem? Otherwise, we will have to figure out a way to get the area_id back from the info we get from UNIX-type shared memory, as that is the only way to build a BBitmap reusing memory allocated by another process. If the UNIX shared memory is working, and you have a pointer to the bitmap data there, it's possible to use area_for() to get the corresponding area_id.

Mmm, didn't think about the fact that the BBitmap *must* be associated with an area. Ah, but yes, looks like area_for could work. Or maybe using Haiku's implementation of shared memory would be better? Well, I guess I'll leave those questions for a later PR.

Reply:
>> Do we get this far in the execution of WebKit yet?
>
> For some reason, no. (Next problem to investigate?)

And now it apparently does get that far. Regardless, I'll wait till the next PR to address this.

Reply:
Actually, no, it doesn't get that far.